Platform/de.lproj/Localizable.strings

```diff
@@ -844,7 +844,7 @@
 "The selected boot image contains the word '%@' but the guest architecture is '%@'. Please ensure you have selected an image that is compatible with '%@'." = "Das gewählte Start-Abbild enthält das Wort „%1$@”, die Gast-Architektur ist jedoch „%2$@”. Bitte stellen Sie sicher, dass Sie ein passendes Abbild für „%3$@” ausgewählt haben.";
 
 /* No comment provided by engineer. */
-"The target does not support hardware emulated serial connections." = "Das Ziel unterstützt keine simulierte Hardware für serielle Anschlüsse.";
+"The target does not support hardware emulated serial connections." = "Das Ziel unterstützt keine Hardware-Emulation für serielle Anschlüsse.";
```

Comment:
❓ While translating "target" with "Ziel" is correct, here "target" may be less suited as "target" may refer to a VM or something else that isn't a literal target to be shot at or goal to be reached.

What does "target" here refer to? Depending on the answer there may be a better German translation.

Reply:
I think target refers to the VM, or a connection to the VM/target.
@osy Do you know what this error message means? Is target just the VM or something else?

Reply:
Maybe "Zielsystem" ist more fitting

Reply:
```suggestion
"The target does not support hardware emulated serial connections." = "Das Zielsystem unterstützt keine Hardware-Emulation für serielle Anschlüsse.";
```

Applied suggestion "Zielsystem".